modules/tokens/utils/types.ts

```diff
@@ -15,11 +15,22 @@ export interface TokenWithSecret extends Token {
 
 export function tokenFromRow(
 	row: prisma.Prisma.TokenGetPayload<any>,
+	hashToOrig: (hash: string) => string,
 ): TokenWithSecret {
 	return {
 		...row,
 		createdAt: row.createdAt.toISOString(),
 		expireAt: row.expireAt?.toISOString() ?? null,
 		revokedAt: row.revokedAt?.toISOString() ?? null,
+		token: hashToOrig(row.tokenHash),
```

Comment:
we should probably split this up in to two functions. one for just the token w/o the secret, then one that accepts a raw string for the token.

we don't need to return the hashed token ever.

Reply:
Done!